make/langtools/src/classes/build/tools/symbolgenerator/CreateSymbols.java

```diff
@@ -2384,7 +2384,10 @@ private boolean readAttribute(ClassFile cf, FeatureDescription feature, Attribut
                 MethodDescription method = (MethodDescription) feature;
                 method.methodParameters = new ArrayList<>();
                 for (MethodParameters_attribute.Entry e : params.method_parameter_table) {
-                    String name = cf.constant_pool.getUTF8Value(e.name_index);
+                    String name = null;
+                    if (e.name_index != 0) {
+                        name = cf.constant_pool.getUTF8Value(e.name_index);
+                    }
```

Comment:
```suggestion
                    String name = e.name_index == 0 ? null
                            : cf.constant_pool.getUTF8Value(e.name_index);
```
assuming a continuation indent of 8.